scripts/ci/test_source.sh

```diff
@@ -4,9 +4,10 @@ set -ex
 # Install CLI & CLI testsdk
 echo "Installing azure-cli-testsdk, azure-cli-core, azure-cli from source code"
 git clone https://github.com/Azure/azure-cli --depth 1
-pip install -e azure-cli/src/azure-cli-testsdk
-pip install -e azure-cli/src/azure-cli-core
-pip install -e azure-cli/src/azure-cli
+pip install -e azure-cli/src/azure-cli-testsdk --no-deps
+pip install -e azure-cli/src/azure-cli-core --no-deps
+pip install -e azure-cli/src/azure-cli --no-deps
```

Comment:
Without adding `azure-cli-telemetry`, it will install from PyPI and cause failure:

```
The conflict is caused by:
    The user requested portalocker==2.3.2
    azure-cli-telemetry 1.0.6 depends on portalocker~=1.2
```

Perhaps we should discover all packages like

https://github.com/Azure/azure-cli/blob/1d065f0cb183ab32fef32c7eeb27882ce03dfaa0/scripts/release/debian/build.sh#L41-L42

```
find azure-cli/src/ -name setup.py -type f | xargs -I {} dirname {} | xargs pip install --no-deps
pip install -r azure-cli/src/azure-cli/requirements.py3.$(uname).txt
```